package.json

```diff
@@ -21,7 +21,7 @@
         "check:types": "ts-node scripts/check-for-typescript.ts tsc --noEmit --pretty",
         "check:license": "ts-node scripts/nrfconnect-license.ts check",
         "prepare": "husky install",
-        "update-publish-script": "tsc scripts/nordic-publish.ts -t es2015 -m node16"
+        "postinstall": "esbuild scripts/nordic-publish.ts --bundle --outfile=scripts/nordic-publish.js --platform=node --log-level=warning"
```

Comment:
Changing to esbuild for performance reasons?

Reply:
Changing because I want to bundle all dependencies (`commander`, `semver`, `shasum`) instead relying that they are present on the machine doing the install in compatible versions.

And `tsc` can only transpile but cannot bundle.